ci/azure/linux.yml

```diff
@@ -27,6 +27,7 @@ jobs:
         CONDA_PY: "36"
         CONDA_ENV: pandas
         TEST_ARGS: "--only-slow --skip-network"
+        LOCALE_OVERRIDE: "zh_CN.UTF-8"
```

Comment:
We have `_locale` in the name of the build, but we weren't setting `LOCALE_OVERRIDE`

Reply:
we should be testing ZH and IT is this the missing one?

Reply:
Not sure which is the build that had to use Italian, but we didn't have one for what I've seen.

Set the locale to `it_IT.UTF8` here, I guess that's what we want.